pkg/result/processors/fixer.go

```diff
@@ -43,218 +53,221 @@ func (p Fixer) Process(issues []result.Issue) ([]result.Issue, error) {
 		return issues, nil
 	}
 
-	outIssues := make([]result.Issue, 0, len(issues))
-	issuesToFixPerFile := map[string][]result.Issue{}
-	for i := range issues {
-		issue := &issues[i]
-		if issue.Replacement == nil {
-			outIssues = append(outIssues, *issue)
-			continue
-		}
-
-		issuesToFixPerFile[issue.FilePath()] = append(issuesToFixPerFile[issue.FilePath()], *issue)
-	}
-
-	for file, issuesToFix := range issuesToFixPerFile {
-		err := p.sw.TrackStageErr("all", func() error {
-			return p.fixIssuesInFile(file, issuesToFix)
-		})
-		if err != nil {
-			p.log.Errorf("Failed to fix issues in file %s: %s", file, err)
+	p.log.Infof("Applying suggested fixes")
```

Comment:
Is this for debugging or intentionally left? I think we're usually pretty sparse with logging other than in error or debug context. No strong opinion, just curious.

Reply:
yes, it is a choice, and this log is not displayed by default.